packages/shell-api/api-extractor.json

```diff
@@ -0,0 +1,57 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+  "mainEntryPointFilePath": "<projectFolder>/lib/api.d.ts",
+  "apiReport": {
+    "enabled": false
+  },
+  "docModel": {
+    "enabled": false
+  },
+  "bundledPackages": [
+    "@mongodb-js/devtools-connect",
+    "@mongodb-js/devtools-proxy-support",
+    "@mongodb-js/oidc-plugin",
+    "@mongosh/arg-parser",
+    "@mongosh/service-provider-core",
+    "@mongosh/types",
+    "mongodb",
+    "bson"
```

Comment:
We have bson set up in mongodb-ts-autocomplete, so I might remove it from here.

Reply:
I removed it from here and then mongodb-ts-autocomplete in devtools-shared replaces the bson import to use its one. That way we can use the same bson shared everywhere (shell-api and mql) and we don't end up extracting all the types again and again. This might premature optimisation, I don't know. But it isn't hard to undo either.